### DISCUSSION TITLE
Intégration de notifications Sentry dans Tchap (Webhook)

### DISCUSSION THREAD

**Discussion by pskl** [betagouv/cap#29] [Besoin]
### Besoin

```text
Il serait très pratique de pouvoir brancher l'apparition de nouvelles issues Sentry (l'instance incubateur) dans un canal Tchap, ça pourrait aider à centraliser l'information pour l'équipe d'un projet.

La même technique pourrait servir à annoncer des déploiements Scalingo par exemple.
```

### Solutions envisageables

```text
- Configurer un webhook dans l'instance Sentry pour Tchap tel que c'est déjà fait pour Mattermost
```

### Informations complémentaires

- [X] J'avais déjà entendu ce besoin au sein de la communauté beta.gouv
- [X] Je me porte volontaire pour participer à la mise en place de la future solution choisie

**Comment by revolunet** [comment]
Hot news : une intégration Tchap <-> https://github.com/matrix-org/matrix-hookshot est prévue

**Reply by pskl** [reply]
Merci de l'info et très cool ! Est-ce qu'on a un ETA sur le projet ?

**Reply by revolunet** [reply]
@NicolasBuquet en sait peut-être plus

**Comment by revolunet** [comment]
A noter que cela fonctionne bien avec un workflow [activepieces](https://activepieces.com)

**Comment by NicolasBuquet** [comment]
Bonjour.
Au vu de l'intérêt croissant de la communauté pour utiliser des webhooks avec Tchap, nous avons abordé le sujet en équipe.

Une solution existe avec matrix-hookshot en effet.

Nous allons essayer d'ajouter cela à notre roadmap déjà très chargée cette année.

Aucune ETA promise, désolé.

Mais je serais très surpris que cela puisse arriver durant le 1er semestre.

Mais soyez assuré que l'on communiquera largement quand ce sera prêt ! 😄

**Comment by revolunet** [comment]
En attendant l'offre hookshot proposée par Tchap, vous pouvez demander des webhooks via l'offre activepieces de beta via le formulaire de "demandes-ops"
